src/VisualStudio/Setup.Next/VisualStudioSetup.Next.csproj

```diff
@@ -47,6 +54,10 @@
       <Name>ServicesVisualStudio.Next</Name>
       <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup%3b</IncludeOutputGroupsInVSIX>
       <ForceIncludeInVSIX>true</ForceIncludeInVSIX>
+      <Ngen>true</Ngen>
```

Comment:
Given we're setting the same thing here everywhere, can we just use an ItemDefinitionGroup to default this for everything all at once?